AccountingSystem/private/DataCache.py

```diff
@@ -43,7 +43,7 @@ def getReportData( self, reportRequest, reportHash, dataFunc ):
     Get report data from cache if exists, else generate it
     """
     reportData = self.__dataCache.get( reportHash )
-    if reportData == False:
+    if reportData is None:
```

Comment:
But the point is that PEP8 advocates the comparison to None if possible. There is a difference between None and 0 for instance. (So now I think it was related to CS values, where 0 would then not be allowed in a specific case)